setup-intel.sh

```diff
@@ -0,0 +1,13 @@
+#! /bin/bash
+
+echo "Make sure you are logged in as cluster admin"
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+oc apply -f "${DIR}/intel/aikit-operator-subscription.yaml"
+oc apply -f "${DIR}/intel/ovms-operator-subscription.yaml"
+
+echo "Waiting for 60s to allow operators to install"
```

Comment:
Missing a wait here

Reply:
you might want to play with `oc wait`
https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#wait

Something like this might work for you:
```
oc wait csv/openvino-operator.v1.0.0 -n openshift-operators --for=jsonpath='{.status.phase}'=Succeeded 
```

Reply:
It does not look like the oc wait command works as the examples would suggest. 

`oc get csv/aikit-operator.v2021.2.102120 -o jsonpath='{.status.phase}'
Succeeded%  `

`oc wait csv/aikit-operator.v2021.2.102120 -n openshift-operators --for=jsonpath='{.status.phase}'=succeeded

error: unrecognized condition: "jsonpath={.status.phase}=succeeded"`

Since the operators reliably install within a minute, currently I am thinking that a `sleep 60` or `sleep 120` should be safe enough to proceed. Unless there's a better way to do it other than `oc wait`

Reply:
regarding my previous comment, the `oc wait` succeeds with oc client version 4.10+